internal/services/v1/experimental_test.go

```diff
@@ -177,23 +179,51 @@ func TestBulkExportRelationships(t *testing.T) {
 		{tf.FolderNS.Name, "owner"},
 		{tf.DocumentNS.Name, "editor"},
 		{tf.FolderNS.Name, "editor"},
+		{tf.DocumentNS.Name, "caveated_viewer"},
```

Comment:
can we add similar cases to the filtered bulkexport test?

Reply:
Yes, I'll add